src/main/java/edu/gonzaga/Hand.java

```diff
@@ -1,32 +1,186 @@
 package edu.gonzaga;
 
 import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Hand {
     private ArrayList<Card> cardList;
     private Integer handLimit;
+    private boolean validHand;
+    private Integer numCardsToCure;
 
-    private void checkHandFollowsLimit() {
-
+    /**
+     * Checks if Hand object follows limit, and sets it to resulting value of t/f
+     * @returns true if hand has less or equal number of objects as the limit, or false otherwise
+     * @author Izzy T 
+     */
+    public boolean doesHandFollowLimit() { 
+        if (cardList.size() > handLimit) {
+            validHand = false;
+        }
+        else {
+            validHand = true;
+        }
+        return validHand;
     }
 
+    /**
+     * DVC for a Hand object - sets the limit to 7, declares it a valid hand to start 
+     * 
+     * @author Izzy T
+     */
     public Hand() {
+        handLimit = 7; // 7 basic/event cards allowed in a Hand 
+        validHand = true;
+        numCardsToCure = 5; // 5 cards of a color to cure cities of that color 
```

Comment:
Can you make another constructor that allows me to set the numCardsToCure to a different variable (for one of the specific roles)